### PR TITLE
Resolve esoteric IDs in the Road Travel

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Road Travel

**What?**

Resolve esoteric IDs in the fuel category

**Why?**

Provide better comparability across EFs

### CODE REVIEW

**Review by AnastasiaLobanova — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -40,14 +40,14 @@ Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_lpg-distanc
...
 Energy,Fuel,fuel_type_aviation_fuel_kerosene_transport,Aviation fuel (Kerosene),kg-CO2e/l,2.62632,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation fuel (Kerosene)/Jet A1. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,combustion
-Energy,Fuel,fuel_type_aviation_gasoline_transport,Aviation gasoline,kg-CO2e/l,2.3123049,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,combustion
+Energy,Fuel,fuel_type_aviation_petrol_transport,Aviation petrol,kg-CO2e/l,2.3123049,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation petrol. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:50]: This change is outside the Road Travel category - likely to cause conflicts!

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -354,7 +354,7 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_lcv-fuel_source_petrol-engine_
...
+Transport,Rail Travel,passenger_train-route_type_commuter_rail-fuel_source_unknown,Commuter rail passenger train,kg-CO2e/pmile,0.143,,EPA,2021,US,2021/08/23,Emission intensity for commuter rail passenger train with no information on inclusion of upstream emissions. This emission factor accounts for commuter rail services between a central city and adjacent suburbs (also called regional rail or suburban rail). It is intended for use in the distance-based method defined in the GHG Protocol Scope 3 Calculation Guidance. If fuel data are available then the fuel-based method should be used. Updated by the EPA in April 2021.,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:357]: This change is outside the Road Travel category - likely to cause conflicts! There are a number of these below...

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:50]: This change still needs to be removed as it clashes with 49

**Review by patstirling — CHANGES_REQUESTED** [review]
Just one change still outside the category!

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: should be resolved now!

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**